### DISCUSSION TITLE
Please relaese keda 2.8.2 with chart 2.8.3 - waiting for support of port override

### DISCUSSION THREAD

**Discussion by hajdukda** [kedacore/charts#336] [Q&A] [answered]
Hi,

I'm just curious when new version will be released, Ive seen the changes Im interested in merged, but I see that they are not yet available in latest chart and image :(

**Comment by tomkerkhove** [comment, marked as answer]
We are going to release KEDA 2.9 in 2 weeks which will include these changes

**Reply by hajdukda** [reply]
Hello any update on the release ?

**Reply by tomkerkhove** [reply]
It's already out - https://github.com/kedacore/charts/pull/341

However, it looks like I forgot to announce it which I will do now

**Reply by tomkerkhove** [reply]
See https://github.com/kedacore/charts/releases/tag/v2.9.0

Sorry about that.

**Reply by hajdukda** [reply]
Have one more question:
Charts.yaml mentions that its possible to deploy on older K8S versions:
"# We allow pre-release versions for cloud-specific Kubernetes versions such as  v1.21.5-gke.1302 or v1.18.9-eks-d1db3c"
But there seem to be no "pre-release" version for EKS ?
Is it possible to make the requirement a bit broader like 1.22+ ?

**Reply by hajdukda** [reply]
Or rather - is there any real reason to deprecate older versions (like API changes or what not) or its an artificial rule to support only X versions N-2 ?

**Reply by hajdukda** [reply]
It can be in readme that running older version is "own responsibility" but not hardcode version requirement in chart :X...

**Reply by tomkerkhove** [reply]
No, we actually don't support it because of the API versions that we rely on.
